l20n/format/ast.py

```diff
@@ -49,7 +85,7 @@ def __init__(self, name):
         self.name = name
 
 
-class Section(Node):
+class Section(Resource):
     def __init__(self, key, body=None, comment=None):
         super(Section, self).__init__()
         self.key = key
```

Comment:
I think it'd make sense to move entities() onto a shared base class of Resource and Section. Also, I'd pass in body and comment to the constructor of the base class.